openslides_backend/action/actions/meeting_user/create.py

```diff
@@ -38,3 +40,33 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 f"MeetingUser instance with user {instance['user_id']} and meeting {instance['meeting_id']} already exists"
             )
         return super().update_instance(instance)
+
+    def get_history_information(self) -> Optional[HistoryInformation]:
+        information = {}
+        for instance in self.instances:
+            instance_information = []
+            if "group_ids" in instance:
+                if len(instance["group_ids"]) == 1:
+                    instance_information.extend(
+                        [
+                            "Participant added to group {} in meeting {}",
+                            fqid_from_collection_and_id(
+                                "group", instance["group_ids"][0]
+                            ),
+                        ]
+                    )
+                else:
+                    instance_information.append(
```

Comment:
This kind of information hides the fact that someone did not allowed things, i.e. could participate on elections, where he is not allowed to vote.

Reply:
What would be your suggestion? Display all groups? That could potentially be a lot - the history information string should only provide a gist of what happened. Details aboout the changed data must be looked up in history mode or, in the future, in the diff view. The history information is not the right place to provide all data.